.github/workflows/release.yml.template

```diff
@@ -27,11 +27,16 @@ jobs:
     name: Build and publish policy
     runs-on: ubuntu-latest
     needs: test
+    permissions:
+      # Required to push to GHCR
+      contents: read
+      packages: write
+      # Required by cosign keyless signing
+      id-token: write
```

Comment:
Interesting 🔍

Reply:
Some background information about that. GH Actions have a a permissive profile by default, which allows operations like push to ghcr using GitHub's token. However, this permissive profile doesn't allow to access the id-token.

As soon as you add something to the `permissions` section the strict profile is used. All the directives added under `permissions` are added to it. As a result of that, I had to specify two extra permissions required to push to ghcr, because they were not granted by the strict profile out of the box.